### ISSUE TITLE
The display is taking too much viewport in mobile

### ISSUE THREAD

**Issue by Daryadi-S** [filegator/filegator#32]
Hai, thank you for the project. This seems the most promising php file manager (been using responsive file manager, tinyfilemanager, and some others) out there.

The display list is taking too much viewport in mobile. I think it's better to use a single line for a file display in mobile. With the current it takes a table of 3 row just to display a file in mobile view.

Thank you 😃

**Comment by alcalbg** [comment]
Yup, it makes sense. The current list is just the table default from here: https://buefy.org/documentation/table

**Comment by kocane** [comment]
@alcalbg is it possible to make the view on mobile more similar to that in a browser, perhaps by hiding some of the columns? 

I feel the cards/box view when going to the site on phones doesn't work as well.

**Comment by kocane** [comment]
@alcalbg any chance you can weigh in?
